.changeset/true-dots-invent.md

```diff
@@ -0,0 +1,6 @@
+---
+"@gradio/audio": minor
```

Comment:
Can we add a story for Audio?

Reply:
It looks like @hannahblair intentionally didn't add since the `Audio` component is about to undergo a pretty big redesign

Reply:
Lol I completely missed that!!